app/bufferless-slugid.js

```diff
@@ -0,0 +1,36 @@
+import { v4 } from '@lukeed/uuid';
+
+/**
+ *
+ * Returns a randomly generated uuid v4 compliant slug which conforms to a set
```

Comment:
Did you code this yourself?

Any intention to send this to the slugid folks to see if they'll include it in their repo and get rid of the Buffer requirement?

Reply:
I did code this myself, yes. I just adapted some of their code, mostly. 

I'd probably be able to make a PR, but right now the uuid library I chose [`@lukeed/uuid`](https://github.com/lukeed/uuid) is fast but not cryptographically secure. A PR would probably entail refactoring internals with [`uuid`](https://www.npmjs.com/package/uuid) as suggested  https://github.com/taskcluster/slugid/issues/12.